content/components.html

```diff
@@ -60,6 +60,46 @@ <h1>
         </nav>
       </header>
     </div>
+
+    <div class=st-container>
+      <header class=profile>
+        <div>
+          <div class=profile-user>
+            <h1 class=username>User name</h1>
```

Comment:
Isn't `.profile-user > h1` enough and you need a `.username`?
Or is it reused elsewhere?